readme.md

```diff
@@ -93,18 +93,25 @@ athloi publish -- --access=public
 
 ### concurrency
 
-A global concurrency option which can be used to execute multiple tasks in parallel. By default only one task will run at a time.
+A global option which will execute up to the given number of tasks concurrently. By default one task will be run at a time.
```

Comment:
Should there be an upper-limit to the number of tasks we can run concurrently? 🤔